cockatrice/src/game/player/player.cpp

```diff
@@ -2031,7 +2031,7 @@ void Player::eventCreateToken(const Event_CreateToken &event)
         return;
     }
 
-    CardItem *card = new CardItem(this, QString::fromStdString(event.card_name()), event.card_id());
+    CardItem *card = new CardItem(this, QString::fromStdString(event.card_name()), QString(), event.card_id());
     // use db PT if not provided in event
```

Comment:
Is this the place to do the defaults we were doing in DeckList?

Reply:
Probably?